data/scripts/talkactions/gm/teleport_to_player.lua

```diff
@@ -0,0 +1,61 @@
+local teleportToCreature = TalkAction("/listplayers")
+
+function teleportToCreature.onSay(player, words, param)
+	-- create log
+	logCommand(player, words, param)
+
+	local onlyActive = param == "active" and true or false
+
+	local players = Game.getPlayers()
+	local playerList = {}
+
+	for _, targetPlayer in ipairs(players) do
+		if targetPlayer:getName() == player:getName() then
+			goto continue
+		end
+
+		if not onlyActive then
+			table.insert(playerList, targetPlayer)
+		else
+			local isGhost = targetPlayer:isInGhostMode()
+			local isTraining = onExerciseTraining[targetPlayer:getId()]
+			local isIdle = targetPlayer:getIdleTime() >= 5 * 60 * 1000
+			local isActive = not isGhost and not isTraining and not isIdle
```

Comment:
now that this in a few different places, we could make a function in player.lua probably